From d5fd845aab4074ed6d0950551e007ea43fd2d0df Mon Sep 17 00:00:00 2001
From: Zhenyu Wang <zhenyu.z.wang@intel.com>
Date: Tue, 19 Aug 2008 07:30:01 +0800
Subject: [PATCH] Fix reverted LVDS bios capability dword definition

Issue found by Peter Clifton.
(cherry picked from commit 0e6759bb6ad7293b6447bd20e3d7d8fdee46aba7)

diff --git a/src/i830_bios.h b/src/i830_bios.h
index a8d9add..2b073f3 100644
--- a/src/i830_bios.h
+++ b/src/i830_bios.h
@@ -152,13 +152,13 @@ struct bdb_lvds_options {
     uint8_t panel_type;
     uint8_t rsvd1;
     /* LVDS capabilities, stored in a dword */
-    uint8_t rsvd2:1;
-    uint8_t lvds_edid:1;
-    uint8_t pixel_dither:1;
-    uint8_t pfit_ratio_auto:1;
-    uint8_t pfit_gfx_mode_enhanced:1;
-    uint8_t pfit_text_mode_enhanced:1;
     uint8_t pfit_mode:2;
+    uint8_t pfit_text_mode_enhanced:1;
+    uint8_t pfit_gfx_mode_enhanced:1;
+    uint8_t pfit_ratio_auto:1;
+    uint8_t pixel_dither:1;
+    uint8_t lvds_edid:1;
+    uint8_t rsvd2:1;
     uint8_t rsvd4;
 } __attribute__((packed));
 
-- 
1.5.6.5

